src/commands/apphosting-backends-list.ts

```diff
@@ -28,7 +28,7 @@ export const command = new Command("apphosting:backends:list")
       );
     }
 
-    const backends = backendRes?.backends;
+    const backends = backendRes.backends ?? [];
```

Comment:
This change feels weird. `backendRes.backends` is non-nullable, so technically that `??` is unreachable

Maybe there's a type issue here.